### ISSUE TITLE
Import top 3 mems

### ISSUE THREAD

**Issue by nicolasmaia** [wilddom/memrise2anki-extension#43]
This extension only imports mems you've already chosen, but it would be more helpful if it imported the top 3 most voted mems and showed them as hints in the Anki cards, so you don't need to check memrise all the time for mems when learning new words.

**Comment by wilddom** [comment]
Getting the top-rated mem should be easy. We could get even more, but there is just one field for mems... do you want multiple mems merged into one field?

Note to self:
https://www.memrise.com/api/mem/get_many_for_thing/?thing_id=[x]&column_a=1&column_b=2

**Comment by picsi** [comment]
>do you want multiple mems merged into one field?

I think that sounds good. 3 is an arbitrary number I thought of, just to give the learner more options. Similar to how Heisig's RTK deck pulls the top 3 kanji mneumonics from koohii.

**Comment by picsi** [comment]
Also, fetching the pictures is a must too :)

**Comment by wilddom** [comment]
The future of mems is still somewhat unclear, memrise even temporarily removed the possibility to see mems created by other users:
https://community.memrise.com/t/memrise-release-notes-27-apr-2017/12072
https://community.memrise.com/t/memrise-release-notes-4-may-2017/12411

And to be honest, I'm quite busy at the moment and I don't really need this feature myself. So, I can't make any promises...

**Comment by picsi** [comment]
That's true. In fact, their mem fuckup drove me away from memrise and towards Anki and this add-on in the first place :) 

Community-created mems are the only advantage Memrise has over Anki, and if this add-on can fetch them, Anki will be superior is every sense. But take your time 🙏

**Comment by Adrian545454** [comment]
Hi are there any updates on this feature? I would really like to download multiple mems for the same word

**Comment by wilddom** [comment]
Mems are no longer accessible
